buildpacks/php/tests/integration/smoke.rs

```diff
@@ -19,6 +23,62 @@ fn smoke_test_bundled_hello_world_app() {
     );
 }
 
+#[test]
+#[ignore = "integration test"]
+fn smoke_test_composer_json_scripts_as_objects() {
+    let temp = tempfile::tempdir().unwrap();
+    let source = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+        .join("tests/fixtures/smoke/hello-world")
+        .canonicalize()
+        .unwrap();
+    let app_dir = temp.path();
+    copy_dir_all(source, app_dir).unwrap();
```

Comment:
`libcnb-test` has built-in functionality for manipulating an existing fixture `app_dir_preprocessor`:
https://docs.rs/libcnb-test/latest/libcnb_test/struct.BuildConfig.html#method.app_dir_preprocessor

I think we should either use that, or else add an explicit fixture for this. (For Python I have a preference of using explicit fixtures over manipulation within the test -- it makes it easier to manually build that fixture outside of the tests etc. Though I don't know what David's preferences are.)